doc/manual/decoding.rst

```diff
@@ -27,6 +27,10 @@ Scikit-learn API enforces the requirement that data arrays must be 2D. A common
 
 To recover the original 3D data, an ``inverse_transform`` can be used. The ``epochs_vectorizer`` is particularly useful when constructing a pipeline object (used mainly for parameter search and cross validation). The ``epochs_vectorizer`` is the first estimator in the pipeline enabling estimators downstream to be more advanced estimators implemented in Scikit-learn. 
 
+Vectorizer
+^^^^^^^^^^
+Scikit-learn API provides functionality to chain transformers and estimators by using :class:`sklearn.Pipeline`. We can apply MNE transformers in scikit-learn pipeline to chain them. Further it becomes useful for cross validation and grid search for parameters. Vectorizer is typically the last step(before the estimator in the pipeline) to convert higher dimensional data into two dimensional, compatible with the estimator.
```

Comment:
'sklearn.Pipeline' => 'sklearn.pipeline.Pipeline'

'Further' => 'Furthermore'

'step(' => 'step ('

'cross validation' => 'cross-validation'
'grid search'  => 'grid-search'

Overall, check your English, the current sentences are not really clear/specific. I would write something like:

"Decoding pipelines may be constructed following Scikit-Learn API. However, whereas Scikit-Learn transformers and estimators generally expect 2D data (`n_samples` * `n_features`), MNE transformers typically output data with higher dimensionality (e.g. `n_samples` * `n_channels` * `n_frequencies` * `n_times`). A `Vectorizer` therefore needs to be applied between the MNE and the Scikit-Learn steps: e.g: `make_pipeline(Xdawn(), Vectorizer(), LogisticRegression())`